conf/configuration.go

```diff
@@ -355,7 +362,7 @@ func updateStorageConfFromClowder(conf *storage.Configuration) {
 	conf.PGPassword = clowder.LoadedConfig.Database.Password
 }
 
-func updateTopicsMapping(c *ConfigStruct) error {
```

Comment:
is it safe to remove this error?

Reply:
I think we added it by habit. The function does not seem to do anything that can return an error, so yes, it's safe IMO.